README.md

```diff
@@ -109,6 +108,59 @@ npm run migrate up
 
 if migrations are not running, or get stuck, please check that you are using node v12
 
+### Snapshot Setup
+
+* `GATSBY_SNAPSHOT_SPACE`: the snapshot space where the proposals will be published
+* `SNAPSHOT_PRIVATE_KEY`, `GATSBY_SNAPSHOT_ADDRESS`: a pair address/key with permissions to publish at that snapshot space
+* The configured SNAPSHOT_PROPOSAL_TYPE for the project is 'single-choice', which means each voter may select only one choice. See [available voting systems](https://docs.snapshot.org/proposals/voting-types#single-choice-voting)
+
+#### Creating a Snapshot space
+
+You are going to need to register an ENS name in the network you'll create the space in.
+For this purpose, you are going to need ETH / Rinkeby ETH / Goerli ETH. Use the faucets to get it:
+
+[Rinkeby Faucet](https://rinkebyfaucet.com/)
+
+[Goerli Faucet](https://goerlifaucet.com/)
+
+Then follow instructions on [Snapshot](https://docs.snapshot.org/spaces/create)
+
+#### Strategy
+
+- DCL Governance uses two strategies:
```

Comment:
These are the strategies used in the production space.
<img width="769" alt="image" src="https://user-images.githubusercontent.com/8242162/188499263-c4276f1f-6eb3-47d8-afb2-7b0656a1a2a0.png">

Reply:
Maybe it is not necessary to add the strategies here 🤔 as it depends on the configuration of each space.

Reply:
I added them all just in case